t/issues/t4891-flux-resource-empty-requires.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
```

Comment:
Question: is the specific issues test required, or is this already covered in `t2350-resource-list.t`? If it is already covered, then I'd vote to just drop this test.

BTW, if you keep this test, then it needs to restore the confguration of the t4000-issues instance, since this could cause subsequent tests to fail. Or, it should run in a subinstance.

Reply:
yeah, its a duplicate.  We can drop.